src/components/AccessControlNew.tsx

```diff
@@ -102,9 +111,11 @@ const AccessControlNew = () => {
         setEditName(false)
         dispatch(ruleActions.setRule({
             Name: '',
+            Description: '',
             Source: [],
             Destination: [],
-            Flow: 'bidirect'
+            Flow: 'bidirect',
+            Disabled: false
```

Comment:
Codacy has a fix for the issue: [Replace `············Disabled:·false` with `········Disabled:·false,`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
        Disabled: false,
```